packages/katello/katello-repos/katello-client.repo

```diff
@@ -7,6 +7,17 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
 enabled=1
 gpgcheck=0
 
+[jortel-gofer]
+name=Copr repo for gofer owned by jortel
+baseurl=https://copr-be.cloud.fedoraproject.org/results/jortel/gofer/epel-7-$basearch/
```

Comment:
This will only work on EL7. It probably needs some macro so this works EL 5, 6 and Fedora as well.

Reply:
Updated